### PR TITLE
Remove line breaks in model generation

### LINKED ISSUE #243: Save breaks with models containing line breaks in text
**Describe the bug**
You cannot save a model containing line breaks in variables or conditions. The error outputs the start of an SVG-File.

**To Reproduce**
1. Create a CEG Model
2. Add a node
3. Rename the variable name to something containing a line break
4. Save
5. See console

**Expected behavior**
The save action does not break.

**Screenshots**

**Additional context**

### PR DESCRIPTION
**Pull request by sebeder**
Closes #243

### CODE REVIEW

**Review by wehrlean — CHANGES_REQUESTED** [review]
For a short time an additional scrollbar is visible and for large models it is even possible to scroll down and see the copied svg. Maybe you can set it to hidden/unvisible. 

If there is not as simple solution, you can still merge it.

**Comment by sebeder** [comment]
The problem is, that the thing *has* to be visible for save-svg-to-png to be rendered. Actually, my first solution was to do the manipulation in a purely virtual svg, but then, the image is just empty.

**Comment by sebeder** [comment]
I think we'll give that to @junkerm to decide what's happening. I'll assign him.

**Comment by wehrlean** [comment]
I thought you already tried it out.
The only thing I could think of right now would be to work with the z-index and maybe hide it right behind the original.

**Comment by sebeder** [comment]
That's ugly, but doable, and better than what we have now, I think. I'll try that. Thanks for the hint!

File: web/src/app/modules/views/main/editors/modules/graphical-editor/services/model-image.service.ts [hunk 1]
```diff
@@ -42,7 +49,10 @@ export class ModelImageService extends Monitorable {
                     factor = factor * 0.95;
                     pngAsBase64 = await saveAsPng.svgAsPngUri(svg, { 'scale': factor, 'height': maxHeight, 'encoderOptions': 1.0 });
                 }
+                console.log(pngAsBase64);
```
Comment (wehrlean) [inline web/src/app/modules/views/main/editors/modules/graphical-editor/services/model-image.service.ts]: Push the commit ;)

File: web/src/app/modules/views/main/editors/modules/graphical-editor/components/graphical-editor.component.html [hunk 2]
```diff
@@ -27,6 +27,7 @@ <h5 class="card-header">{{name}}
                     [ngStyle]="{'background': 'url(/assets/img/grid.png)', 'background-size': editorGridButton.getBackgroundSize()}">
                 </div>
             </div>
+            <div id="renderingContainer"></div>
```
Comment (wehrlean) [inline web/src/app/modules/views/main/editors/modules/graphical-editor/components/graphical-editor.component.html:30]: At the moment it is above the original, thus the dots are visible for the time of saving. Put it under the original

image

**Review by wehrlean — CHANGES_REQUESTED** [review]
See comments

**Review by wehrlean — APPROVED** [review]

**Pull request merged**